lib/config/cli.js

```diff
@@ -22,8 +22,16 @@ function getConfig(argv) {
 
   const coersions = {
     boolean: val => val === 'true',
-    list: val =>
-      val === '[]' || val === '' ? [] : val.split(',').map(el => el.trim()),
+    list: val => {
+      if (val === '[]' || val === '') {
```

Comment:
looks totally fine, but if I'm not mistaken, this check is not needed anymore then, as `[]` is caught by the `JSON.parse` and returns `[]` correctly, and the `''` throws an error `Unexpected end of input` so if falls back to `''.split(',')` which resolves to `[]` as well, so ll26-28 can be removed imo

Reply:
I decided to leave as-is, as otherwise it needs an extra test or two added to verify it. I think that `''` won't work as you expect. I might just remove the `[]` one.

Reply:
Alright, just wanted to keep the code clean
I can also add the two test-cases if you want me to. Otherwise this looks good-to-go to me